client/lib/analytics/utils/current-user.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Internal dependencies
+ */
+import hashPii from './hash-pii';
+
+/**
+ * Module variables
+ */
+let _currentUser;
+
+/**
+ * Gets current user.
+ *
+ * @returns {Ojbect|undefined} Current user.
+ */
+export function getCurrentUser() {
+	return _currentUser;
+}
+
+/**
+ * Sets current user.
+ *
+ * @returns {Ojbect|undefined} Current user.
+ */
+export function setCurrentUser( { ID, username, email } ) {
+	const userData = { ID: parseInt( ID, 10 ), username, email };
+
+	if ( ! userData.ID || isNaN( userData.ID ) || ! userData.username || ! userData.email ) {
+		return; // Invalid user data.
+	}
+
+	userData.hashedPii = {
```

Comment:
@MicBosi Note this new approach to compiling hashed PII for marketing analytics.

Reply:
Example use in `ad-tracking.js` looks like this.

```
const currentUser = getCurrentUser();
const params = currentUser ? { em: currentUser.hashedPii.email } : {};
```